src/SnackbarItem/SnackbarItem.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect, useRef } from 'react';
 import clsx from 'clsx';
-import { withStyles, WithStyles, createStyles, Theme, emphasize } from '@material-ui/core/styles';
+import { withStyles, WithStyles, createStyles } from '@material-ui/styles';
+import { createMuiTheme, Theme, emphasize } from '@material-ui/core/styles';
```

Comment:
Should it be `utils/createTheme.js` instead of `createMuiTheme`?

Reply:
Good catch!

Reply:
```suggestion
import { Theme, emphasize } from '@material-ui/core/styles';
import createTheme from '../utils/createTheme';
```